cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -678,29 +687,33 @@ class HydratorPlusPlusTopPanelCtrl {
     };
     this.previewLoading = true;
     this.loadingLabel = 'Stopping';
+    let pipelinePreviewPlaceholder = 'The preview of the pipeline';
+    const pipelineName = this.HydratorPlusPlusConfigStore.getName();
+    if (pipelineName.length > 0) {
```

Comment:
we could also do something like 

const pipelinePreviewPlaceholder = `The preview of the pipeline${pipelineName.length > 0 ? ' pipelineName' : ''}`